src/cljs/reagent_toolbox/core.cljs

```diff
@@ -55,19 +55,39 @@
 
 (def card-title (reagent/adapt-react-class (.-CardTitle js/ReactToolbox)))
 
-(def checkbox (reagent/adapt-react-class (.-Checkbox js/ReactToolbox)))
+(def checkbox-component (reagent/adapt-react-class (.-Checkbox js/ReactToolbox)))
+
+(defn checkbox
+  [properties]
```

Comment:
I've seen it both ways. I tend to lean on consistency so I go with arguments on their own line regardless of count. But it's a cosmetic issue and I can adjust to whatever we decide is the de facto standard for us.